### PR TITLE
🆕(site) Add New Theme Feature phosphor

### PR DESCRIPTION
**Pull request by austindeh**
**Phosphor theme**

Styling changes for Phosphor theme.

** Needs to be merged with latest components**

- Resolves #

### CODE REVIEW

File: site/sass/components/_nav.phosphor.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  &__secondary {
+    border: solid 2px var(--blue-850);
+    // margin: 0 20px;
+
+    @include minLangMQ($bkpt-map--header--large) {
+      margin: 0 10px;
+      width: calc(100vw - 40px);
+    }
```
Comment (Snugug) [inline site/sass/components/_nav.phosphor.scss:35]: Non-color items changing

File: site/sass/components/_header.phosphor.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Site Header
+
+.header {
+  border: solid var(--blue-850) 2px;
+  margin: 20px;
+  width: calc(100vw - 40px);
```
Comment (Snugug) [inline site/sass/components/_header.phosphor.scss:6]: Non-color items changing

File: site/sass/components/_card-routing.phosphor.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  &__footer {
+    margin-bottom: 2rem;
+
+    @include mq($card-2-col) {
+      margin-bottom: 1rem;
+      margin-top: auto;
+    }
+  }
```
Comment (Snugug) [inline site/sass/components/_card-routing.phosphor.scss:77]: Non-color items changing

File: site/sass/components/_card-routing.phosphor.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+  &__header {
+    margin-bottom: 1rem;
+  }
+
+  &__title {
+    @include mq($grid-3-col) {
+      max-width: 12ch;
+    }
+  }
```
Comment (Snugug) [inline site/sass/components/_card-routing.phosphor.scss:58]: Non-color items changing

File: site/sass/components/_card-routing.phosphor.scss [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+  &__body {
+    color: var(--grey-700);
+    margin-bottom: 2rem;
+    max-width: 34ch;
+
+    @include mq($card-2-col) {
+      margin-bottom: 1rem;
+    }
+  }
```
Comment (Snugug) [inline site/sass/components/_card-routing.phosphor.scss:68]: Non-color items changing

File: site/sass/components/_card-routing.phosphor.scss [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  &__content {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    @container card-routing (min-width: 504px) {
+      align-items: start;
+      text-align: start;
+    }
+  }
```
Comment (Snugug) [inline site/sass/components/_card-routing.phosphor.scss:48]: Non-color items changing

File: site/sass/components/_card-routing.phosphor.scss [hunk 7]
```diff
@@ -0,0 +1,78 @@
+.card-routing {
+  $clamp-check: clamp(1px, 1vw, 2px);
+  $scale-max: 1600px;
+  $grid-3-col: 1058px;
+  $card-2-col: 631px;
+  $grid-2-card-1-col: 698px;
+
+  background: var(--white);
+  border: solid 2px var(--blue-850);
+  border-radius: 0.625rem;
```
Comment (Snugug) [inline site/sass/components/_card-routing.phosphor.scss:10]: non-color item changing

File: site/sass/components/_card-routing.phosphor.scss [hunk 8]
```diff
@@ -0,0 +1,78 @@
+.card-routing {
+  $clamp-check: clamp(1px, 1vw, 2px);
+  $scale-max: 1600px;
+  $grid-3-col: 1058px;
+  $card-2-col: 631px;
+  $grid-2-card-1-col: 698px;
+
```
Comment (Snugug) [inline site/sass/components/_card-routing.phosphor.scss:7]: Items tied to non-color items

File: site/sass/components/_body.phosphor.scss [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+//   100% {
+//     transform: translatey(-50%);
+//   }
+// }
+// @keyframes scanlines {
+//   to {
+//     opacity: 0.6;
+//   }
+// }
```
Comment (Snugug) [inline site/sass/components/_body.phosphor.scss:75]: Remove commented code

**Review by Snugug — COMMENTED** [review]
Thanks for this work! I've got a couple of immediate questions:
* There are a bunch of places where non-color styles are changing (marked a bunch). Why is that? This should be a color swap, not redeclaring things like paddings and widths and sizings
* There are a bunch of places where a color (background or text) is redeclared using a CSS variable where it would have used one to begin with. It looks like, in `_colors.phosphor.scss`, you're redeclaring all of the CSS variables and overriding them with phosphor colors. Shouldn't the updated colors cascade in as expected?
* We shouldn't commit the `.vscode` settings
* `components/colors.phosphor` and `components/body.phosphor` should be imported like the rest of the phosphor components instead of handling the `[data-theme]` on their own.
* I'd almost prefer a `font-family: $font-mono !important` on the phosphor body rather than redeclaring all of the font families in the `type` extend
* The expand icon for our sidebar can't be seen, and is in the wrong place, in this theme:
image
* As is the "On this page" (it'd also maybe be nice to keep the thin line on the left)
image
* I think it's important to keep the distinction between a filled CTA and a hollow CTA in this theme, for instance in the cookie consent banner

File: site/sass/components/_body.phosphor.scss (hunk 9 above)
Comment (monkishtypist) [reply site/sass/components/_body.phosphor.scss:75]: updated/removed. there was some testing around adding CRT static styling, but abandoned for time. This was a remnant of that.

File: site/sass/components/_card-routing.phosphor.scss (hunk 7 above)
Comment (monkishtypist) [reply site/sass/components/_card-routing.phosphor.scss:10]: removed

File: site/sass/components/_card-routing.phosphor.scss (hunk 8 above)
Comment (monkishtypist) [reply site/sass/components/_card-routing.phosphor.scss:7]: removed

File: site/sass/components/_card-routing.phosphor.scss (hunk 6 above)
Comment (monkishtypist) [reply site/sass/components/_card-routing.phosphor.scss:48]: removed

File: site/sass/components/_card-routing.phosphor.scss (hunk 5 above)
Comment (monkishtypist) [reply site/sass/components/_card-routing.phosphor.scss:68]: removed

File: site/sass/components/_card-routing.phosphor.scss (hunk 4 above)
Comment (monkishtypist) [reply site/sass/components/_card-routing.phosphor.scss:58]: removed

File: site/sass/components/_card-routing.phosphor.scss (hunk 3 above)
Comment (monkishtypist) [reply site/sass/components/_card-routing.phosphor.scss:77]: removed

File: site/sass/components/_nav.phosphor.scss (hunk 1 above)
Comment (monkishtypist) [reply site/sass/components/_nav.phosphor.scss:35]: The nav specifically had inset/margin added in design for added impact (1/2)

File: site/sass/components/_header.phosphor.scss (hunk 2 above)
Comment (monkishtypist) [reply site/sass/components/_header.phosphor.scss:6]: The nav specifically had inset/margin added in design for added impact (2/2)

**Comment by monkishtypist** [comment]
@Snugug I refactored a bunch of stuff to make this theme way more independent structurally, in case you need to tweak files later on. I also addressed most of your feedback (inline) with a few comments called out below.


> There are a bunch of places where non-color styles are changing [...]

This was accidental/missed code from being removed, however for the nav this was intentional for a specific design touch

> There are a bunch of places where a color (background or text) is redeclared using a CSS variable where it would have used one to begin with. It looks like, in _colors.phosphor.scss, you're redeclaring all of the CSS variables and overriding them with phosphor colors. Shouldn't the updated colors cascade in as expected?

Unfortunately not all colors cascaded in all places as intended for the theme, so in some components specifically these had to be updated.

> I'd almost prefer a font-family: $font-mono !important on the phosphor body rather than redeclaring all of the font families in the type extend

I tested this in the body component but oddly the !important didn't take and the other declared fonts remained... checked the console to see the important declaration crossed out?

> The expand icon for our sidebar can't be seen, and is in the wrong place, in this theme:

This is an image so needs to be updated at that level. Defering for now so I can get some sleep...

File: site/sass/components/_nav.phosphor.scss (hunk 1 above)
Comment (monkishtypist) [reply site/sass/components/_nav.phosphor.scss]: I actually tweaked this to be more of a color swap and removed the margins. this resolved mobile issues that were missed.

**Comment by Snugug** [comment]
Thanks! I'm going to take a look now. As for the expand icon, it's an inline SVG so it can be recolored with `fill`

**Review by Snugug — APPROVED** [review]

**Review by Snugug — APPROVED** [review]

**Pull request merged**